### ISSUE TITLE
This extension is not installable on any currently installed products

### ISSUE THREAD

**Issue by rpopeszku** [wixtoolset/issues#5577]
I just finished installing Visual Studio Community 2017. 
Microsoft Visual Studio Community 2017 
Version 15.1 (26403.7) Release
VisualStudio.15.Release/15.1.0+26403.7
Microsoft .NET Framework
Version 4.6.01586
Installed Version: Community

I have a working Visuals Studio 2015 solution that has a WIX project named Setup. When I opened this solution in 2017 it loads properly but the Setup project says (incompatible).

So I tried to install the "WIX Toolset Visual Studio 2017 Extension" at this link 
https://marketplace.visualstudio.com/items?itemName=RobMensching.WixToolsetVisualStudio2017Extension

But I get an error message window that says "This extension is not installable on any currently installed products." 
The expanded detail is...
5/9/2017 9:05:16 AM - Microsoft VSIX Installer
5/9/2017 9:05:16 AM - -------------------------------------------
5/9/2017 9:05:16 AM - Initializing Install...
5/9/2017 9:05:16 AM - Extension Details...
5/9/2017 9:05:16 AM - 	Identifier         : WixToolset.VisualStudioExtension.Dev15
5/9/2017 9:05:16 AM - 	Name               : Wix Toolset Visual Studio 2017 Extension
5/9/2017 9:05:16 AM - 	Author             : .NET Foundation and contributors
5/9/2017 9:05:16 AM - 	Version            : 0.9.21.62588
5/9/2017 9:05:16 AM - 	Description        : Visual Studio integration for the WiX Toolset - the most powerful set of tools available to create your Windows installation experience.
5/9/2017 9:05:16 AM - 	Locale             : en-US
5/9/2017 9:05:16 AM - 	MoreInfoURL        : http://wixtoolset.org/
5/9/2017 9:05:16 AM - 	InstalledByMSI     : False
5/9/2017 9:05:16 AM - 	SupportedFrameworkVersionRange : [4.0,)
5/9/2017 9:05:16 AM - 
5/9/2017 9:05:16 AM - 	SignatureState     : Unsigned
5/9/2017 9:05:16 AM - 	Supported Products : 
5/9/2017 9:05:16 AM - 		Microsoft.VisualStudio.Community
5/9/2017 9:05:16 AM - 			Version : [15.0,16.0)
5/9/2017 9:05:16 AM - 
5/9/2017 9:05:16 AM - 	References         : 
5/9/2017 9:05:16 AM - Signature Details...
5/9/2017 9:05:16 AM - 	Extension is not signed.
5/9/2017 9:05:16 AM - 
5/9/2017 9:05:16 AM - Searching for applicable products...
5/9/2017 9:05:16 AM - Found installed product - Microsoft Visual Studio Community 2015
5/9/2017 9:05:16 AM - Found installed product - Microsoft Visual Studio 2015 Shell (Integrated)
5/9/2017 9:05:16 AM - Found installed product - Global Location
5/9/2017 9:05:16 AM - VSIXInstaller.NoApplicableSKUsException: This extension is not installable on any currently installed products.
   at VSIXInstaller.App.InitializeInstall(Boolean isRepairSupported)
   at VSIXInstaller.App.InitializeInstall()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()

What am I doing wrong?
Rudy Popeszku

**Comment by rpopeszku** [comment]
Turns out I was incorrectly trying to install the extension. Apparently it will not work if you just download and run the extension directly from the link I posted above. You must install this extension within Visual Studio by going to the Tools -> Extensions & Updates -> Updates -> Visual Studio Marketplace  and then search for the WIX extension above and install it that way.
Installing Visual Studio extensions is not something I do on a regular basis and there are no details on the website as to how to properly install the extension so I was confused.
